servicetest/networkgateway/test_network_gateway.py

```diff
@@ -110,13 +110,18 @@ def test_network_policy_drop(self):
         """
         try:
             sc = Container()
-            sc.start(DATA)
-            sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
-            sc.launch_command("python " +
+            success = sc.start(DATA)
+            assert success is True
+            
+            success = sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
+            assert success is True
+
+            pid, success = sc.launch_command("python " +
```

Comment:
In all places where "pid" will not be used, replace it with a "_" (an underscore).